tests/conftest.py

```diff
@@ -1,8 +1,99 @@
 # -*- coding: utf-8 -*-
+import base64
+import os
+import pickle
 import pytest
 import re
+import zlib
 
 from scrapinghub.hubstorage.serialization import MSGPACK_AVAILABLE
+from scrapinghub import HubstorageClient
+from scrapinghub.legacy import Connection
+
+
+DEFAULT_PROJECT_ID = "2222222"
+DEFAULT_ENDPOINT = 'http://storage.vm.scrapinghub.com'
+DEFAULT_DASH_ENDPOINT = 'http://33.33.33.51:8080/api/'
+
+
+TEST_PROJECT_ID = os.getenv('HS_PROJECT_ID', DEFAULT_PROJECT_ID)
+TEST_SPIDER_NAME = 'hs-test-spider'
+TEST_FRONTIER_SLOT = 'site.com'
+TEST_BOTGROUP = 'python-hubstorage-test'
+TEST_COLLECTION_NAME = "test_collection_123"
+TEST_AUTH = os.getenv('HS_AUTH', 'f' * 32)
+TEST_ENDPOINT = os.getenv('HS_ENDPOINT', DEFAULT_ENDPOINT)
+TEST_COLLECTION_NAME = "test_collection_123"
+TEST_ADMIN_AUTH = os.getenv('AUTH', 'f' * 32)
+TEST_USER_AUTH = os.getenv('USER_AUTH', 'e' * 32)
+TEST_DASH_ENDPOINT = os.getenv('DASH_ENDPOINT', DEFAULT_DASH_ENDPOINT)
+
+
+class VCRGzipSerializer(object):
+    """Custom ZIP serializer for VCR.py."""
+
+    def serialize(self, cassette_dict):
+        # receives a dict, must return a string
+        # there can be binary data inside some of the requests,
+        # so it's impossible to use json for serialization to string
+        cassette_dict = normalize_cassette(cassette_dict)
+        compressed = zlib.compress(pickle.dumps(cassette_dict, protocol=2))
+        return base64.b64encode(compressed).decode('utf8')
+
+    def deserialize(self, cassette_string):
+        # receives a string, must return a dict
+        decoded = base64.b64decode(cassette_string.encode('utf8'))
+        return pickle.loads(zlib.decompress(decoded))
+
+
+def normalize_endpoint(uri, endpoint, default_endpoint):
+    old = endpoint
```

Comment:
We could replace the function with one-liner, wdyt?
```
def normalize_endpoint(uri, endpoint, default_endpoint): 
    return uri.replace(endpoint.rstrip('/'), default_endpoint.rstrip('/'))
```

Reply:
Sure.

Reply:
Done.